README.md

```diff
@@ -380,6 +380,27 @@ const usernameInputElement = getByTestId(container, 'username-input')
 > `data-testid`s from the blog post
 > ["Making your UI tests resilient to change"][data-testid-blog-post]
 
+### `getByRole`
```

Comment:
Let's put this above `getByTestId`. I want the docs to read from most recommended to least. I would recommend `getByRole` over `getByTestId` when it applies.

Reply:
Done